community/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/ClosingIteratorTest.scala

```diff
@@ -37,8 +37,8 @@ class ClosingIteratorTest extends CypherFunSuite {
 
   test("should not close prematurely") {
     //Given
-    val wrapee   = Iterator(Map("k" -> intValue(42)))
-    val iterator = new ClosingIterator(wrapee, taskCloser, exceptionDecorator)
+    val wrappee   = Iterator(Map("k" -> intValue(42)))
```

Comment:
Please fix alignment.

Reply:
Will do. It isn't a real word afaict, but per language rules wrappee is the proper spelling based on wrapper.